packages/core/http/core-http-router-server-internal/src/router.ts

```diff
@@ -236,7 +247,15 @@ export class Router<Context extends RequestHandlerContextBase = RequestHandlerCo
     this.patch = buildMethod('patch');
   }
 
-  public getRoutes({ excludeVersionedRoutes }: InternalGetRoutesOptions = {}) {
+  public static on(event: RouterEvents, cb: (req: CoreKibanaRequest) => void) {
+    Router.ee.on(event, cb);
```

Comment:
nit: `emitEven` is easier to read than `ee`, which one has to interpret.

Reply:
abreviating `EventEmitter` to `ee` is a common programming practice i think its fine to keep it